src/Spryker/Zed/EventBehavior/Business/Model/EventResourcePluginResolver.php

```diff
@@ -119,11 +120,11 @@ protected function mapPluginsByResourceName(): void
      */
     protected function getEffectivePlugins(array $resources): array
     {
-        $effectivePlugins = [];
-        if (empty($resources)) {
+        if ($resources === []) {
```

Comment:
We have a similar moment in our [code styles](https://documentation.spryker.com/guidelines/code_style_guide.htm)
```suggestion
        if (empty($resources)) {
```

or you can use variant: `if (count($resources) === 0) { ... }`

Reply:
I prefer direct explicits comparison, `count` is indirect, why do you need to count it to determine if it's empty or not, It doesn't actually get count, it's still O(1), also `empty` is not type specific while the variable `$resources` is always an array.